src/include/common/cpu/rpp_cpu_rgb_hsv_conversion.hpp

```diff
@@ -0,0 +1,227 @@
+    pMask[0] = _mm256_castsi256_ps(_mm256_cmpeq_epi32(pxIntH, avx_px2));                                            // case 2:
+    pVecR = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecR), _mm256_and_ps(pMask[0], pA));                           //     rf = p;
+    pVecG = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecG), _mm256_and_ps(pMask[0], pV));                           //     gf = v;
+    pVecB = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecB), _mm256_and_ps(pMask[0], pS));                           //     bf = t; break;
+    pMask[0] = _mm256_castsi256_ps(_mm256_cmpeq_epi32(pxIntH, avx_px3));                                            // case 3:
+    pVecR = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecR), _mm256_and_ps(pMask[0], pA));                           //     rf = p;
+    pVecG = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecG), _mm256_and_ps(pMask[0], pH));                           //     gf = q;
+    pVecB = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecB), _mm256_and_ps(pMask[0], pV));                           //     bf = v; break;
+    pMask[0] = _mm256_castsi256_ps(_mm256_cmpeq_epi32(pxIntH, avx_px4));                                            // case 4:
+    pVecR = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecR), _mm256_and_ps(pMask[0], pS));                           //     rf = t;
+    pVecG = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecG), _mm256_and_ps(pMask[0], pA));                           //     gf = p;
+    pVecB = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecB), _mm256_and_ps(pMask[0], pV));                           //     bf = v; break;
+    pMask[0] = _mm256_castsi256_ps(_mm256_cmpeq_epi32(pxIntH, avx_px5));                                            // case 5:
+    pVecR = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecR), _mm256_and_ps(pMask[0], pV));                           //     rf = v;
+    pVecG = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecG), _mm256_and_ps(pMask[0], pA));                           //     gf = p;
+    pVecB = _mm256_or_ps(_mm256_andnot_ps(pMask[0], pVecB), _mm256_and_ps(pMask[0], pH));                           //     bf = q; break;}
+}
+
+#else
+
+// Converts RGB to HSV color space using SSE vectorization, processes 4 pixels simultaneously
+inline void rgb_to_hsv(__m128 &pVecR, __m128 &pVecG, __m128 &pVecB, __m128 &pH, __m128 &pS, __m128 &pV, __m128 &pAdd)
+{
+    __m128 pMask[4], pDelta;
+    // Convert RGB to HSV
+    pV = _mm_max_ps(pVecR, _mm_max_ps(pVecG, pVecB));                                                               // cmax = RPPMAX3(rf, gf, bf);
+    pS = _mm_min_ps(pVecR, _mm_min_ps(pVecG, pVecB));                                                               // cmin = RPPMIN3(rf, gf, bf);
+    pDelta = _mm_sub_ps(pV, pS);                                                                                    // delta = cmax - cmin;
+    pH = xmm_p0;                                                                                                    // hue = 0.0f;
```

Comment:
same as previous comment for pH and pV assignment

Reply:
Fixed